src/app/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -44,60 +47,73 @@ import { NavigateToFolder } from '../../../store/actions';
   changeDetection: ChangeDetectionStrategy.OnPush,
   host: { class: 'aca-search-results-row' }
 })
-export class SearchResultsRowComponent implements OnInit {
+export class SearchResultsRowComponent implements OnInit, OnDestroy {
   private node: MinimalNodeEntity;
+  private onDestroy$ = new Subject<boolean>();
 
   @Input()
   context: any;
 
-  constructor(private store: Store<AppStore>) {}
+  name$ = new BehaviorSubject<string>('');
+  title$ = new BehaviorSubject<string>('');
 
-  ngOnInit() {
-    this.node = this.context.row.node;
-  }
+  constructor(
+    private store: Store<any>,
+    private alfrescoApiService: AlfrescoApiService
+  ) {}
 
-  get name() {
-    return this.getValue('name');
-  }
-
-  get title() {
-    return this.getValue('properties["cm:title"]');
+  ngOnInit() {
+    this.updateValues();
+
+    this.alfrescoApiService.nodeUpdated
+      .pipe(takeUntil(this.onDestroy$))
+      .subscribe((node: any) => {
+        const row = this.context.row;
+        if (row) {
+          const { entry } = row.node;
+
+          if (entry.id === node.id) {
+            entry.name = node.name;
+            entry.properties = Object.assign({}, node.properties);
+
+            this.updateValues();
+          }
+        }
+      });
   }
 
-  get description() {
-    return this.getValue('properties["cm:description"]');
-  }
+  private updateValues() {
+    this.node = this.context.row.node;
 
-  get modifiedAt() {
-    return this.getValue('modifiedAt');
-  }
+    this.name$.next(this.node.entry.name);
 
-  get size() {
-    return this.getValue('content.sizeInBytes');
+    const title = this.node.entry.properties['cm:title'];
+    this.title$.next(title ? `( ${title} )` : '');
   }
 
-  get user() {
-    return this.getValue('modifiedByUser.displayName');
+  ngOnDestroy() {
+    this.onDestroy$.next(true);
+    this.onDestroy$.complete();
   }
 
-  get hasDescription() {
-    return this.description;
+  get description(): string {
+    return this.node.entry.properties['cm:description'];
   }
 
-  get hasTitle() {
-    return this.title;
+  get modifiedAt(): Date {
+    return this.node.entry.modifiedAt;
   }
 
-  get showTitle() {
-    return this.name !== this.title;
+  get size(): number {
+    return this.node.entry.content.sizeInBytes;
```

Comment:
I think content is missing for nodes of type folder... and also description might

Reply:
I've updated the code, thanks.